WordPress/Classes/Services/PeopleService.swift

```diff
@@ -68,6 +68,25 @@ struct PeopleService {
         })
     }
 
+    /// Loads a page of Viewers associated to the current blog, starting at the specified offset.
+    ///
+    /// - Parameters:
+    ///     - offset: Number of records to skip.
+    ///     - count: Number of records to retrieve. By default set to 20.
+    ///     - success: Closure to be executed on success.
+    ///     - failure: Closure to be executed on failure.
+    ///
+    func loadViewersPage(offset: Int = 0, count: Int = 20, success: ((retrieved: Int, shouldLoadMore: Bool) -> Void), failure: (ErrorType -> Void)? = nil) {
+        remote.getViewers(siteID, offset: offset, count: count, success: { viewers, hasMore in
+            self.mergePeople(viewers)
```

Comment:
Again, haven't tested in instruments, but it seems like we could have a retain cycle here:

`self` -(strong)-> `remote` -(strong)-> `self`